tests/integration/kubernetes/confidential_kbs.sh

```diff
@@ -321,6 +321,7 @@ kbs_k8s_svc_host() {
 			local host
 			host=$(kubectl get nodes -o jsonpath='{.items[0].status.addresses[?(@.type=="ExternalIP")].address}' -n "$KBS_NS")
 			[ -z "$host"] && host=$(kubectl get nodes -o jsonpath='{.items[0].status.addresses[?(@.type=="InternalIP")].address}' -n "$KBS_NS")
+			[[ "$(hostname -i)" =~ "$host" ]] && host="localhost"
```

Comment:
It should be capital `I` to check against all host IP addresses, not just the first one

Reply:
I ended up dropping this commit.